src/main/user-impl/java/com/mysql/cj/jdbc/ha/ca/plugins/DefaultMonitorService.java

```diff
@@ -47,6 +52,7 @@ public DefaultMonitorService(Log log) {
             new BasicConnectionProvider(),
             hostInfo,
             propertySet,
+            propertySet.getIntegerProperty(PropertyKey.monitorDisposeTime).getValue(),
```

Comment:
Something doesn't look right if you have to pass the whole property set, and then pass a single entry of this set again?

If the code being called get other values from the property set, can't it get this value the same way?
Or if that's the only value being fetched from the property set, why passing the whole set as parameter?